resources/store/index.js

```diff
@@ -83,27 +83,6 @@ module.exports = new Vuex.Store( {
 	 * Getters are like computed properties for Vuex state
 	 */
 	getters: {
```

Comment:
As the app shapes up, moving some of this data out of Vuex seems like the best approach. We don't need to keep track of current image or suggestions at a global level any more, it can all happen in the ImageCard component. When the user publishes suggestions, that will dispatch an action containing their choices as part of the payload.